fix(country): fix chapter list by country page

Splaktar opened the pull request:
upgrade maps to latest angular-google-maps
fix some styling issues

Fixes #65.

Before:
![screen shot 2015-11-21 at 9 03 04 pm](https://cloud.githubusercontent.com/assets/3506071/11321621/50099504-9095-11e5-88ae-41b7dc9e3ca9.png)

After
![screen shot 2015-11-21 at 9 03 15 pm](https://cloud.githubusercontent.com/assets/3506071/11321623/51be9e80-9095-11e5-8939-566765b09c84.png)

Splaktar committed:
fix(country): fix chapter list by country page

upgrade maps to latest angular-google-maps
fix some styling issues

tasomaniac commented:
Looks good. One comment: I think there is no point having markers on the smaller maps. I think it will look better with just maps.

Splaktar commented:
@tasomaniac I agree. Someone can click on the chapter to open its page and see a map with a marker if they want.

Splaktar committed:
update(country): Don't put markers on the smaller chapter maps

from code review feedback
Someone can click the chapter to open its page and see a marker if needed

Splaktar commented:
![screen shot 2015-11-21 at 9 25 55 pm](https://cloud.githubusercontent.com/assets/3506071/11321675/a5148b5c-9096-11e5-8ed9-273ab720a5bb.png)
